text/0012-asynchronous-updates.md

```diff
@@ -0,0 +1,151 @@
+* Title: Asynchronous Updates
+* Start Date:
+* Specification PR: https://github.com/meilisearch/specifications/pull/12
+* MeiliSearch Issue:
+
+# Asynchronous updates
+
+## 1\. Feature Description and Interaction
+
+### Summary
+
+This spec is the spec of a feature already in place. 
+
+It's scope is:
+- The name for update list and update identifiers. 
```

Comment:
I personally don't really like the name `update` to mean the actions/tasks that are done by MeiliSearch. I know from the core side, this is an update of the DB, but from the user side, I see more a task than an update.
Plus, the "update" word **as a noun** is really confusing with the widely used word "update" **as a verb**! It's not convenient for naming during development.
Despite this, I'm not sure this would be a good move to change the name of our `updates`.

Reply:
I'm agree, I'll ask the others.

Reply:
There are also possible alternatives: \`action\`, \`process\`, \`job\`, \`work\`, \`operation\`...